MonkeyBot/Modules/Polls/PollMessageUpdater.cs

```diff
@@ -0,0 +1,92 @@
+﻿using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.Entities;
+using MonkeyBot.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MonkeyBot.Modules
+{
+    public class PollMessageUpdater
+    {
+        private DiscordMessage _pollMessage;
+        private DiscordMessageBuilder _pollPreviewMessage;
+        private DiscordEmbedBuilder _pollPreviewMessageEmbed;
+
+        private PollMessageUpdater(CommandContext ctx, DiscordEmbedBuilder pollPreviewMessageEmbed,
+            DiscordMessage pollMessage)
+        {
+            _pollMessage = pollMessage;
+            _pollPreviewMessageEmbed = pollPreviewMessageEmbed;
+
+            _pollPreviewMessage = new DiscordMessageBuilder()
+                .WithEmbed(_pollPreviewMessageEmbed.Build());
+        }
+
+        public DiscordMessage Message => _pollMessage;
+
+        public static string BuildDescription(IEnumerable<PollAnswer> pollAnswers) =>
+            string.Join("\n",
+                pollAnswers.Select(ans => $"{ans.Emoji}`[{ans.Count}]` {ans.Value}"));
+
+        public static IEnumerable<DiscordActionRowComponent>
+            BuildAnswerButtons(IEnumerable<PollAnswer> pollAnswers) =>
+            pollAnswers
+                .GroupBy(x => x.OrderNumber / 5)
+                .Select(g => new DiscordActionRowComponent(g.Select(x => new DiscordButtonComponent(
+                    ButtonStyle.Primary, x.Id, string.Empty,
```

Comment:
Acutally, buttons ids need to be unique only within message. They don't need to have unique identifiers between different messages. I have tested if there is some problem when there are two polls at the same time and I didn't have any issues.

Reply:
Hmm then maybe the problem lies elsewhere. It also works for me initially but if I restart the Bot after I set up both polls I consistently get an interaction failed on the second poll, while the first one still works

Reply:
I think problem lies here 
![image](https://user-images.githubusercontent.com/22114978/139557410-f940037f-5236-44b7-af22-aad9431f93b3.png)
It has to be change to Task.WhenAll instead of awaiting each invocation of StartPollAsync